packages/server/src/standalone/index.ts

```diff
@@ -1,14 +1,14 @@
 import type { WithRequired } from '@apollo/utils.withrequired';
-import { json } from 'body-parser';
+import bodyParser from 'body-parser'; // note that importing 'json' directly doesn't work in ESM
```

Comment:
Essentially, but since we don't own body-parser it's harder to fix :)
This one was the motivation for me to make the smoke test, because it was a runtime failure.